org.sugarlabs.Words.json

```diff
@@ -54,7 +54,7 @@
                 {
                     "type": "git",
                     "url": "https://github.com/sugarlabs/gst-plugins-espeak.git",
-                    "commit": "b8e815a1b8ca1468d92ab8631779e3ee3d832613"
+                    "commit": "7f6e41274fb833a487a7ee8ac0c236f0821330cc"
```

Comment:
Is there any way to provide a data checker here?

Reply:
We can just do something like
```json
{
            "name": "gst-plugins-espeak",
            "no-parallel-make": true,
            "sources": [
                {
                    "type": "git",
                    "url": "https://github.com/sugarlabs/gst-plugins-espeak.git"
                }
            ]
        }
}
```
This will fetch the latest commit always.

Reply:
We can't specify that way or the build won't be reproducible.

Reply:
I see that the repo does have tags https://github.com/sugarlabs/gst-plugins-espeak/tags, let's use them

Reply:
Last release was 6 years ago, should we use it?

Reply:
ouch, no, let's stick with the commit then.

Reply:
@sparshg can you ask Ibiam or James if we can make a release of that plugin?

Reply:
I opened an issue here for this: https://github.com/sugarlabs/gst-plugins-espeak/issues/6

Reply:
I've made a new release.